src/features/layers_in_area/readme.md

```diff
@@ -0,0 +1,37 @@
+## Layer in area
+
+This feature manage all layers that intersecting area of interest (a.k.a focused geometry).
```

Comment:
This feature manages what layers are available in area of interest (a.k.a focused geometry).

^ Is it better explanation?

Reply:
Yeah i think 'intersecting' is very specific word where's this feature manages all related to the AOI layers

Reply:
good point, i will use vkozio variant